src/public_key.rs

```diff
@@ -277,4 +276,85 @@ mod tests {
         );
         assert_eq!(public_key.to_string(), B58.to_string())
     }
+
+    use std::collections::hash_map::DefaultHasher;
+    use std::hash::{Hash, Hasher};
+
+    fn default_bytes() -> [u8; 33] {
+        [
```

Comment:
Note that we use hex! and a const in every other unit test to provide a binary key form. Could you update that?